src/router/index.ts

```diff
@@ -27,6 +28,14 @@ const router: Router = createRouter({
       name: 'Policy',
       component: Policy,
     },
+    {
+      path: '/analytics',
+      name: 'Analytics',
+      component: Analytics,
+      meta: {
+        requiresAuth: true,
```

Comment:
If you could modify this to be something like `requiresWhitelist`, I think that'd be more ideal than throwing an error due to firestore rules.